package.json

```diff
@@ -40,21 +40,22 @@
     "postpublish": "npm run docs:push"
   },
   "peerDependencies": {
-    "@angular/common": "^4.2.6 || ^5.0.0 || ^6.0.0",
-    "@angular/core": "^4.2.6 || ^5.0.0 || ^6.0.0"
+    "@angular/common": "^4.2.6 || ^5.0.0 || ^6.0.0 || ^7.0.0",
```

Comment:
I guess we need to do the same for angular-parse...
angular 7 was tested with aot build on external project?